### PR TITLE
Migrate Java 8 dependencies to Java 11

### PR DESCRIPTION
**Pull request by JackPGreen**
Some of the dependencies are using Java-8 specific dependency versions - now Hazelcast is on Java 11, this is no longer required and the vanilla versions can be used.

### CODE REVIEW

**Review by TomaszGaweda — APPROVED** [review]

**Comment by TomaszGaweda** [comment]
What a silly mistake by me, sorry! Thanks for taking care of it :)

File: hazelcast-sql/pom.xml [hunk 1]
```diff
@@ -476,10 +476,6 @@
                     <groupId>org.jetbrains.kotlin</groupId>
                     <artifactId>kotlin-stdlib</artifactId>
                 </exclusion>
-                <exclusion>
```
Comment (TomaszGaweda) [inline hazelcast-sql/pom.xml:479]: This is an exclusion, not dependency - I think you didn't meant to remove exclusion (= re-adding dependency)

**Review by TomaszGaweda — CHANGES_REQUESTED** [review]
Sorry, missed that exclusion removal = dependency re-adding

File: hazelcast-sql/pom.xml (hunk 1 above)
Comment (JackPGreen) [reply hazelcast-sql/pom.xml]: 7929dcf09751dfb2c823c774d90104a29ae5e187

**Review by TomaszGaweda — APPROVED** [review]

**Pull request merged**